.github/workflows/ci.yml

```
@@ -54,12 +42,63 @@ jobs:
       - name: Run integration tests
         if: matrix.task == 'test:integration'
         run: bun run test:integration
+
+  ci-e2e:
+    runs-on: ubuntu-24.04
+    services:
+      postgres:
+        image: postgres:17
+        env:
+          POSTGRES_PASSWORD: mypassword
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

コンテナ内でしか5432ポートが空いていないのでGitHub Actionのランナーにフォワードしてあげる必要がありそうです。
ref: https://docs.github.com/ja/actions/tutorials/use-containerized-services/use-docker-service-containers#example-mapping-redis-ports
```suggestion
          --health-retries 5
      ports:
          - 5432:5432
```